<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceCryptWindow</class>
 <widget class="QMainWindow" name="FaceCryptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceCryptWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget 
{
background-color: rgb(47, 47, 47);
}</string>
   </property>
   <widget class="QComboBox" name="availableCameras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pixeloid Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGraphicsView" name="camView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QWidget" name="maskOutline" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>291</width>
      <height>193</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="borderLine" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 2px solid white; border-radius : 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pixeloid Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>CAMERA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>borderLine</zorder>
   <zorder>availableCameras</zorder>
   <zorder>camView</zorder>
   <zorder>maskOutline</zorder>
   <zorder>cameraLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
